containers/datalab/Dockerfile.in

```diff
@@ -116,8 +116,7 @@ RUN pip install protobuf==3.0.0b2.post2 && \
     rm tensorflow-0.8.0-cp27-none-linux_x86_64.whl
```

Comment:
Line 82 should be updated to:
`tools/google-cloud-sdk/bin/gcloud config set --installation \`

See #816 and #817.